forte/data/batchers.py

```diff
@@ -144,6 +145,132 @@ def default_configs(cls) -> Dict[str, Any]:
         raise NotImplementedError
 
 
+class FixedSizeDataPackBatcherWithExtractor(ProcessingBatcher):
+    def __init__(self, cross_pack: bool = True):
+        super().__init__(cross_pack=cross_pack)
+        self.instance_pool: List[Annotation] = []
```

Comment:
Also, explain in detail what these variable are doing.

Reply:
Got it.